docs/source/usage/workflows/addLaser.rst

```diff
@@ -22,13 +22,24 @@ The source is applied at a generating surface that is a boundary of an axis-alig
 The implementation is based on the total field/scattered field formulation described in detail :ref:`here <model-TFSF>`.
 A user defines positioning of this box in the total domain in :ref:`incidentField.param <usage-params-core>`.
 
+With properly chosen sources, the generated waves only propagate inwards the volume bounded by the generation surface and there is no considerable leakage or noise.
+The generating surface is otherwise transparent for all signals, such as reflections of lasers from a target.
+A typical setup includes field absorber located outside of the surface (directly or at a distance) for these waves.
```

Comment:
```suggestion
A typical setup includes field absorber located outside of the surface (directly or at a distance).
```
What waves?

Reply:
I meant all other signals as in previous sentence . But i think it's already clear and `these waves` can be just dropped?

Reply:
yes, I would just drop it. But you could also reformulate it somehow. And I think it should be "a field absorber".